### PR TITLE
[KOGITO-5851] - Build and deploy Quarkus from source files cannot be …

### PR DESCRIPTION
**Pull request by spolti**
…accessed once running on Openshift
See: https://issues.redhat.com/browse/KOGITO-5851

Signed-off-by: spolti <fspolti@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: tests/features/kogito-builder.feature [hunk 1]
```diff
@@ -58,6 +58,15 @@ Feature: kogito-builder image tests
       | request_method  | POST                                                                                             |
       | content_type    | application/json                                                                                 |
       | request_body    | {"Driver": {"Points": 2}, "Violation": {"Type": "speed","Actual Speed": 120,"Speed Limit": 100}} |
+    And check that page is served
```
Comment (radtriste) [inline tests/features/kogito-builder.feature:61]: could we add the same on rhpam ?

**Review by radtriste — COMMENTED** [review]
See my comment.

Also I would add the same test feature to Spring Boot as well: check that the [actuator endpoints](https://github.com/kiegroup/kogito-operator/blob/main/core/kogitoservice/probe.go#L35-L36) are responding. wdyt ?

File: tests/features/kogito-builder.feature (hunk 1 above)
Comment (spolti) [reply tests/features/kogito-builder.feature:61]: sure, will add.

Comment (spolti) [reply tests/features/kogito-builder.feature:61]: > See my comment.
> 
> Also I would add the same test feature to Spring Boot as well: check that the [actuator endpoints](https://github.com/kiegroup/kogito-operator/blob/main/core/kogitoservice/probe.go#L35-L36) are responding. wdyt ?

Sure, will add as well.

**Comment by ricardozanini** [comment]
@radtriste this one blocks https://github.com/kiegroup/kogito-operator/pull/1002?

**Review by radtriste — APPROVED** [review]

**Review by vaibhavjainwiz — APPROVED** [review]

**Comment by radtriste** [comment]
corrected image should be available by today EOD

**Comment by radtriste** [comment]
> @radtriste this one blocks [kiegroup/kogito-operator#1002](https://github.com/kiegroup/kogito-operator/pull/1002)?

There was a problem in the builder image for source files. That is why the test was failing. once new image is available, it should be ok

**Pull request merged**